.gitignore

```diff
@@ -5,4 +5,5 @@ dist/
 .eslintcache
 
 # Test output:
-.nyc_output
\ No newline at end of file
+.nyc_output
+.DS_Store
```

Comment:
Can you move this out of the `test output` section? Only the nyc file is related to test output.